recipe/meta.yaml

```diff
@@ -21,13 +21,15 @@ build:
 
 requirements:
   build:
-    - python >=3.5
+    - {{ compiler('c') }}
```

Comment:
Not sure why this should be added. If I depend on Cython for a runtime dependency, then Cython should ensure a compiler is install via its conda recipe.  I shouldn't need that redundancy here.